packages/pluggableWidgets/datagrid-web/src/Datagrid.editorPreview.tsx

```diff
@@ -96,5 +95,5 @@ export function preview(props: DatagridPreviewProps): ReactElement {
 }
 
 export function getPreviewCss(): string {
-    return require("./ui/Datagrid.scss") + require("./ui/DatagridPreview.scss");
```

Comment:
Are the styles (datagrid + datagrid filters) that live in the module (DataGrid 2) loaded and effective in preview mode in S.Pro?

Reply:
Yes

Reply:
> the same compiled scss is injected in design mode (Diego)

👍